# Instant recombination polymerase chain reaction method for constructing chimeric proteins

### Author information
    Mineo Shibasaka, Izumi C. Mori, Shigeko Utsugi, Shaila Shermin Tania, Maki Katsuhara
    Institute of Plant Science and Resources, Okayama University, 2-20-1, Chuo, Kurashiki 710-0046, Japan
    Contacts: ICM, imori@okayama-u.ac.jp; SU, utsugi@okayama-u.ac.jp; MK, kmaki@okayama-u.ac.jp

### Introduction
A variety of techniques are available for introducing site-directed mutations into cloned genes, constructing chimeric genes, or vector substitution. However, selecting the appropriate method and preparing the required materials require both knowledge and technical proficiency, tailored to the specific experimental objectives. In this paper, we introduce a novel, timesaving approach with PCR-based plasmid amplification and <i>Dpn</i>I-based template plasmid removal that is broadly applicable to molecular biological experiments, including mutagenesis, chimera generation, and vector substitution, enabling these tasks to be accomplished with a single versatile technique. 

### Materials and Methods
Aiming at a construction of the chimeric protein possessing the loop C sequence of AtPIP2;1 in the AtPIP2;3 sequence, we performed a PCR-based recombination reaction with the plasmids bearing cDNA sequences of AtPIP2;1 and AtPIP2;3 (AGI codes: At3g53420 and At2g37180).

- Plasmid preparation<br>
The plasmids bearing cDNA sequences of AtPIP2;1 and AtPIP2;3 in pXβG-ev1 plasmid (Preston et al., 1992), pXβG-AtPIP2;1 and pXβG-AtPIP2;3, were purified from <i>E. coli</i> DH5α with the plasmid miniprep kit (QIAprep® Spin Miniprep kit, Qiagen, Hilden Germany) according to the manufacturer’s protocol. pXβG-AtPIP2;1 was used as the DNA fragment donor and pXβG-AtPIP2;3 was used as the recipient plasmid.

- Primer designing<br>
Four primers (#1, #2, #3, and #4) were designed as shown in Table 1. Each primer contains region matching the nucleotide sequence neighbouring the boundary of the chimera as illustrated in Figure 1.

- PCR condition and <i>Dpn</i>I digestion<br>
The reaction mixture (10 µL) consisted of 0.2 µL of 1 unit µL<sup>–1</sup> KOD FX<sup>a</sup> (Toyobo Co. Ltd., Osaka), 5 µL of 2x PCR buffer for KOD FX, 2 µL of 2 mM dNTPs, 20 ng of donor plasmid, 30 ng of recipient plasmid, 0.15 µL of 10 µM of each primers I, II, III and IV. The reaction mixture was fill up with distilled water.

    PCR was performed 19 cycles of 98°C for 10 s, 55°C for 5s and 68°C for 5 min<sup>b</sup>, following a heating at 94°C for 2 min. After the PCR reaction, the methylated recipient plasmids were digested by adding 0.2 µL of <i>Dpn</i>I at 37°C for 1 hours (Fig. 1).

- Cloning of the target plasmid and nucleotide sequencing<br> 
The target plasmid without filling nicks in the DNA strand were introduced into <i>E. coli</i> DH5α cell by a conventional heat shock transformation (Inoue et al., 1990). Cloned plasmid in a single colony was purified by the plasmid miniprep kit and subjected to nucleotide sequence by Sanger method.

### Results
Nucleotide sequencing by Sanger method showed that 4 plasmids out of 4 picked <i>E. coli</i> colonies carried correctly recombined plasmid sequence (Fig. 2).

### Discussion
- Recipient DNA<br>
In the “Instant recombination” protocol, a circular plasmid DNA is used as the recipient. No digestion with restriction enzyme of vector plasmid is required. This does not mean that linearization of the recipient DNA hampers the instant recombination. Linear template can also successfully produce the target plasmid.

- Donor DNA<br>
One can use a PCR-amplified DNA fragment and circular plasmid DNA as the donor. The recombination occurs where the experimenter designed primers without almost no constraint. A multiple annealing position may hamper an expected reaction. 

- DNA polymerase, <i>Dpn</i>I digestion and ligation<br>
Although, we used the KOD-family DNA polymerase in this reaction, any thermotolerant DNA polymerase, which amplify blunt-end product can be used. DNA polymerase, which amplify adenosine-overhanging DNA fragment cannot synthesize proper target products. The digestion with the methylation-sensitive restriction enzyme, <i>Dpn</i>I, secures the contamination of template plasmid upon the cloning. Virtually, we receive nearly 100% of certainty of the successful recombination in our experience. In addition, no ligation procedure is required, since nicked plasmid produced by PCR will be repaired in <i>E. coli</i> cells after the transformation. The cleanup of the PCR product is not necessary. <i>Dpn</i>I-treated reaction mixture is capable for a standard transformation procedure, directly. A blunt-end high fidelity DNA polymerase, a methylation-sensitive restriction enzyme and a general thermal cycler are not specialized products in the molecular biology. No special equipment or kit is need to perform the instant recombination PCR.

- Required quality of oligonucleotide primers<br>
In our experience, no special grade oligonucleotides were required. We use a desalted grade of oligonucleotide for instant recombination PCR.

- Other applications<br>
Not only swapping a region of interest, but the instant recombination PCR procedure can be used for the other application as listed below.
i.	Site directed mutagenesis
ii.	Multi-nucleotide insertion / deletion
iii.	Fusion PCR
iv.	Subcloning of PCR fragment to a cloning vector or any target vector.

- Future improvement<br>
The current protocol shown in this document is not applicable for producing a concatemer of DNA fragments. By future improvement of protocols and primer designing, it may be possible to connect 3 or more pieces/templates to produce the target plasmid in one reaction.

#### Notes
a. KOD FX is not required for this procedure. We have succeeded to obtain the expected target plasmid using KOD polymerase. Any blunt-end high-fidelity thermostable DNA polymerase can be applicable.

b. The PCR conditions should be optimized according to the specific DNA polymerase and annealing temperature of the primers. The composition of the reaction mixture may likewise be modified based on the DNA polymerase used.

### References
- Inoue, H., Nojima H., Okayama, Y. (1990) High efficiency transformation of <i>Escherichia coli</i> with plasmids. Gene 96(1): 23–28.
- Preston , G.M., Carroll, T.P, Guggino, W.B., Agre, P. Appearance of water channels in Xenopus oocytes expressing red cell CHIP28 protein. Science 256: 385–387.

<hr>

<b>Table 1. Nucleotide sequences of primers used for the construction</b><br>
| Primers | 5' to 3' |
| --- | --- |
| #1 | <span style="color: red; ">GCATTTCAA</span><span style="color: blue; ">AGCTCTTACTACACCCG</span> |
| #2 |  <span style="color: red; ">GGTGCCT</span><span style="color: blue; ">GTGCTGTAGCCATCG</span> |
| #3 | <span style="color: blue; ">GATGGCTAC</span><span style="color: red; ">AATACAGGCACCGGA</span> |
| #4 | <span style="color: blue; ">GTAGTAAGAGCT</span><span style="color: red; ">TTGAAATGCCTTCACAAAC</span> |

Blue, nucleotide sequences from the donor. Red, nucleotide sequences from the recipient.

<hr>

